webpack.config.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
         // version: './src/version.js',
     },
     output: {
-        filename: './client-dist/[name].js',
+        path: path.resolve(__dirname, 'client-dist'),
+        filename: '[name].js',
```

Comment:
why undo your fix?

Reply:
vicksonzero (him/Web/Unity) Today at 4:27 PM
why undo your fix for the webpack config?
 
William Chong Today at 4:27 PM
because the correct fix should be publicPath: '/client-dist/',
 
vicksonzero (him/Web/Unity) Today at 4:27 PM
oh
 
William Chong Today at 4:27 PM
original one will create a dist/client-dist/bundle.js when build in prod